docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md

```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
 
 # Uniswap v2 Contract Deployments
 
-Contract addresses for the [Uniswap V2 Factory](https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol) 
+Contract addresses for the [`Uniswap V2 Factory`](https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol) and the [`Uniswap V2Router02`](https://github.com/Uniswap/v2-periphery/blob/master/contracts/UniswapV2Router02.sol)
 
-| Network                                              | Contract Address                                |
-| ---------------------------------------------------- | ----------------------------------------------- |
-| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f`    |
-| Arbitrum                                             | `0xf1D7CC64Fb4452F05c498126312eBE29f30Fbcf9`    |
-| Avalanche                                            | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
-| BNB Chain                                            | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6`    |
-| Base                                                 | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6`    |
-| Optimism                                             | `0x0c3c1c532F1e39EdF36BE9Fe0bE1410313E074Bf`    |
-| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
+| Network                                              | Factroy Contract Address                     | V2Router02 Contract Address                  |
+| ---------------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f` | `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D` |
+| Etherum Sepolia                                      | `0xB7f907f7A9eBC822a80BD25E224be42Ce0A698A0` | `0x425141165d3DE9FEC831896C016617a52363b687` |
+| Arbitrum                                             | `0xf1D7CC64Fb4452F05c498126312eBE29f30Fbcf9` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| Avalanche                                            | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| BNB Chain                                            | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| Base                                                 | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6` | `0x4752ba5dbc23f44d87826276bf6fd6b1c372ad24` |
+| Optimism                                             | `0x0c3c1c532F1e39EdF36BE9Fe0bE1410313E074Bf` | `0x4A7b5Da61326A6379179b40d00F57E5bbDC962c2` |
+| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C` | `0xedf6066a2b290C185783862C7F4776A2C8077AD1` |
```

Comment:
I think we should also add blast.

Reply:
I was planning on waiting till live in the interface, but happy to give devs a running head start... I do need to verify that the contracts are finalized, but I have it ready to go. If I can get verification soon I can add or just go with this and add when verified they are good to go